src/serlio/util/Utilities.cpp

```diff
@@ -286,9 +283,9 @@ std::wstring getProcessTempDir(const std::wstring& prefix) {
 }
 
 time_t getFileModificationTime(const std::wstring& p) {
-	std::wstring pn = p;
 
 #ifdef _WIN32
+	std::wstring pn = p;
```

Comment:
consider merging the two consecutive `_WIN32` blocks here

Reply:
done